src/fancyprint/printer.py

```diff
@@ -284,63 +178,60 @@ def print(self, text: str, align="center", blank_character=" ", left_delimiter=T
 
                 # Make spacing variables
-                space_left = blank_character * amount_of_space
-                space_right = blank_character * amount_of_space
-            elif align == "right":
+                space_left = config.blank_character * amount_of_space
+                space_right = config.blank_character * amount_of_space
+            elif config.align == Align.RIGHT:
                 # Amount of space is the screen width - the side separator's length - the text length
                 amount_of_space = int((terminal_size_h - 2) - length_of_text)
 
-                space_left = blank_character * (amount_of_space - 1)
+                space_left = config.blank_character * (amount_of_space - 1)
                 space_right = " "
-            elif align == "left":
+            elif config.align == Align.LEFT:
                 # Amount of space is the screen width - the side separator's length - the text length
                 amount_of_space = int((terminal_size_h - 2) - length_of_text)
 
                 space_left = " "
-                space_right = blank_character * (amount_of_space - 1)
+                space_right = config.blank_character * (amount_of_space - 1)
 
             if len(space_left) + len(space_right) + length_of_text <= terminal_size_h:
-                if align == "center":
-                    space_right = space_right + blank_character * (
+                if config.align == Align.CENTER:
+                    space_right = space_right + config.blank_character * (
                                 (terminal_size_h - 2) - (len(space_left) + len(space_right) + length_of_text))
-                elif align == "right":
-                    space_left = space_left + blank_character * (
+                elif config.align == Align.RIGHT:
+                    space_left = space_left + config.blank_character * (
                                 (terminal_size_h - 2) - (len(space_left) + len(space_right) + length_of_text))
-                elif align == "left":
-                    space_right = space_right + blank_character * (
+                elif config.align == Align.LEFT:
+                    space_right = space_right + config.blank_character * (
                                 (terminal_size_h - 2) - (len(space_left) + len(space_right) + length_of_text))
 
-            result = f"{Fore.CYAN}|{Fore.RESET}" + space_left + ftext + space_right + f"{Fore.CYAN}|"
+            result = f"{Fore.CYAN}|{Fore.RESET}" + space_left + finalized_text + space_right + f"{Fore.CYAN}|"
 
-            finalized_text_buffer[finalized_text_buffer.index(ftext)] = result
+            finalized_text_buffer[finalized_text_buffer.index(finalized_text)] = result
 
-        if back_separator:
+        if config.back_separator:
             self.separate_line("back")
 
         # Print text in finalized text buffer
         for msg in finalized_text_buffer:
             print(msg)
 
```

Comment:
same here, one would have to change configuration for not having a separator, like if i make a main menu,
i would have to:
\# print line with both separators
\# change config to remove back separator
\# print more lines
\# if some error logger then change config again to block front separator till errors are gone and change config again to get front separator to display again
\# print errors
\# change config to display no front sep to do input...
I might be wrong, but I think it would be efficient to have parameters in this case with default values